spec/helpers/order_helper_spec.rb

```diff
@@ -1,8 +1,15 @@
 require 'rails_helper'
 
+# PRO: Tests with rspec, great job!
+# CON: Could test also if each individual item has the
+# correct discounted price. The total discount may be
+# very likely to already confirm that but not necessarily.
+#
+# For example: 23 + 7 == 13 + 17
```

Comment:
yes, i believe i need to write more tests to this api, sorry about that, i need to write test to check individual discount and unit tests to check total_descounted_itens and every unit test for each single method, if i do it, it will be a great cover of the tests

Reply:
It would improve the test reliability, which basically means that if something changes in the calculation for each individual item we would be warned by a failing test. 👍🏽